sechub-administration/src/test/java/com/daimler/sechub/domain/administration/project/ProjectUpdateAdministrationRestControllerMockTest.java

```diff
@@ -104,6 +112,49 @@ public Void answer(InvocationOnMock invocation) {
 		  verifyNoInteractions(mockedProjectUpdateWhiteListService);
 		/* @formatter:on */
 	}
+	
+	@Test
+	public void when_validator_marks_no_errors___calling_update_project_metadata_calls_update_service_and_returns_http_200() throws Exception {
+
+		/* execute + test @formatter:off */
+        this.mockMvc.perform(
+        		post(https(PORT_USED).buildUpdateProjectMetaData(projectId)).
+        		contentType(MediaType.APPLICATION_JSON_VALUE).
+        		content("{\"metaData\":{\"key1\":\"value1\",\"key2\":\"value2\"}}")
+        		)./*andDo(print()).*/
```

Comment:
Please remove the commented code.

Reply:
Can do. I adopted this from already existing Tests in this and other classes and thought this was left on purpose to give a hint for fast debugging.

@de-jcup would it be ok for you if I remove this `andDo(print())` statements?

Reply:
This was something I did when I started creating restdoc tests. So it is very easy to have communication printed out - suitable for creating and fixing rest doc tests. You just remove the comment parts and you got the output...

And I copied the comment... often ... and so did @4w31ss3 

I think we should either accept this one or remove all inside the rest doc tests.

Reply:
In this case keep them.

Reply:
Team decided to remove all /*andDo(print()).*/-comments with search and replace.

Reply:
removed